Sources/PostgreSQL/SQL/PostgreSQLBinaryOperator.swift

```diff
@@ -65,7 +65,16 @@ public struct PostgreSQLBinaryOperator: SQLBinaryOperator, Equatable, Expressibl
 
     /// See `SQLBinaryOperator`.
     public static let isNot: PostgreSQLBinaryOperator = "IS NOT"
+    
+    /// See `SQLBinaryOperator`.
+    public static let contains: PostgreSQLBinaryOperator = "@>"
 
+    /// See `SQLBinaryOperator`.
+    public static let containsBy: PostgreSQLBinaryOperator = "<@"
```

Comment:
Maybe we should name this one `isContainedBy` since that's what Postgres' docs call it?

https://www.postgresql.org/docs/9.1/functions-array.html

Reply:
Yeah sounds good!